prisma/schema.prisma

```diff
@@ -14,19 +14,31 @@ datasource db {
 }
 
 model User {
-  id String @id @default(uuid())
-  email String @unique
-  firstName String
-  lastName String
-  password String
+  id          String  @id @default(uuid())
+  email       String  @unique
+  firstName   String
+  lastName    String
+  password    String
   avatarImage String?
 
   games Game[]
 }
 
 model Game {
-  id String @id @default(uuid())
+  id   String @id @default(uuid())
   name String
 
   owners User[]
 }
+
+model Screening {
+  id    String   @id @unique @default(uuid())
+  date  DateTime
+  time  DateTime
+  seats Seats[]
+}
+
+model Seats {
+  id        String      @id @unique @default(uuid())
```

Comment:
Please add another field, like name. This should be strings like "A1", "A2", "F4", ...